interface.go

```diff
@@ -55,12 +55,55 @@ type Interface interface {
 	Stats() (*Stats, error)
 }
 
+type Datastore interface {
+	// NewBatch starts a new batch operation that is specific to the datastore implementation
+	NewBatch() interface{}
+
+	// CommitBatch commits a batch that is specific to the datastore implementation
+	CommitBatch(batch interface{}) error
+
+	// CloseBatch closes a batch that is specific to the datastore implementation
+	CloseBatch(batch interface{}) error
+
+	// GetValue returns a value associated with the value key
+	GetValue(valKey []byte) (*Value, error)
+
+	// GetValueKeys returns value keys associated with the multihash
+	GetValueKeys(multihash multihash.Multihash) ([][]byte, bool, error)
```

Comment:
Maybe introduce a type a alias for value keys?
Pros: clearer interface
Cons: value keys mean different things in different indexes

Reply:
Might be useful if a `valueKeys` type contained type information, or was an interface for an underlying concrete type.